ynput/ayon-core
# Max & Maya: Make sure validators being shown in the Publisher UI when they set to be optional in AYON setting

## Changelog Description
This PR is to make sure validators being shown correctly in the Publisher UI when they are being set to be optional in AYON setting.

## Additional info
n/a

## Testing notes:
1. Launch Max/Maya
2. Create Instances
3. Toggle the validators optional switch in Max/Maya settings
4. Check to see whether the validators are being shown in the Publisher UI
